modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/instance/SharedRocksDbInstanceCreator.java

```diff
@@ -197,10 +199,10 @@ private ColumnFamilyOptions createCfOptions(byte[] cfName, Path path) {
                 return add(new ColumnFamilyOptions());
 
             case PARTITION:
-                return add(new ColumnFamilyOptions().useFixedLengthPrefixExtractor(PartitionDataHelper.ROW_PREFIX_SIZE));
+                return add(defaultCfOptions().useCappedPrefixExtractor(PartitionDataHelper.ROW_PREFIX_SIZE));
```

Comment:
What's the difference?

Reply:
If I understood the C++ code correctly, FixedLengthExtractor does not enable prefix seek if a given prefix's length is less than configured (e.g. when we do seek on a table ID + partion ID) only. I can't say that it makes a difference, though (at least I haven't noticed it)